--- setup.orig.py	2015-04-21 11:06:27.490536259 +0000
+++ setup.py	2015-04-21 11:07:28.287654493 +0000
@@ -150,7 +150,7 @@
     ext_modules = []
     libraries = []
 elif sys.platform == 'win32':
-    libraries = ['geos']
+    libraries = ['geos_c']
 else:
     libraries = ['geos_c']
 
@@ -181,7 +181,7 @@
         "shapely.speedups._speedups",
         ["shapely/speedups/_speedups.c"],
         libraries=libraries,
-        include_dirs=[get_config_var('INCLUDEDIR')],),
+        include_dirs=[],),
 ]
 
 cmd_classes = setup_args.setdefault('cmdclass', {})
